src/io/flutter/console/FlutterConsoleFilter.java

```diff
@@ -67,48 +68,76 @@ public FlutterConsoleFilter(@NotNull Module module) {
     this.module = module;
   }
 
+  @VisibleForTesting
+  @Nullable
+  public VirtualFile fileAtPath(@NotNull String pathPart) {
+
+    // lib/main.dart:6
```

Comment:
I do not understand this comment.

Oh! that's an example. The quotes on line 134 helped. See also line 115.

Reply:
😄   

I'll add more quotes.